lib/CppInterOp/CppInterOp.cpp

```diff
@@ -860,38 +936,47 @@ TCppScope_t GetParentScope(TCppScope_t scope) {
   return (TCppScope_t)P;
 }
 
-TCppIndex_t GetNumBases(TCppScope_t klass) {
-  auto* D = (Decl*)klass;
-
+static TCppIndex_t GetNumBases(compat::Interpreter& I,
```

Comment:
Why we have these clones of their counterparts taking `I`. I mean I know why but why only a handful of them and not all?